org.adempiere.base/src/org/compiere/model/MPackageExp.java

```diff
@@ -75,5 +75,35 @@ protected boolean afterDelete (boolean success)
 		return false;
 	 return true;
 	}	//	afterDelete
-	
+
+	/**
+	 * @param from the source MPackageExp to copy the details
+	 * @return
+	 */
+	public int copyDetailsFrom(MPackageExp from) {
+		int count = 0;
+		for (MPackageExpDetail fromDetail : from.getDetails()) {
+			MPackageExpDetail toDetail = new MPackageExpDetail(getCtx(), 0, get_TrxName());
+			PO.copyValues(fromDetail, toDetail, getAD_Client_ID(), getAD_Org_ID());
+			toDetail.setAD_Package_Exp_ID(getAD_Package_Exp_ID());
+			toDetail.saveEx();
+			count++;
+		}
+		return count;
+	}	//	copyDetailsFrom
+
+	/**
+	 * Get the details of a packout definition
+	 * @return array of details
+	 */
+	private List<MPackageExpDetail> getDetails() {
+		final String where = "AD_Package_Exp_ID=?";
+		final String orderBy = "Line, AD_Package_Exp_ID";
+		List<MPackageExpDetail> list = new Query(getCtx(), MPackageExpDetail.Table_Name, where, get_TrxName())
+				.setParameters(getAD_Package_Exp_ID())
+				.setOrderBy(orderBy)
+				.list();

```

Comment:
intentional to copy inactive lines too ?

Reply:
you're right - added the filter for inactive